src/Http/Http.Abstractions/test/HttpResponseWritingExtensionsTests.cs

```diff
@@ -31,6 +31,15 @@ public async Task WritingText_MultipleWrites()
             Assert.Equal(22, context.Response.Body.Length);
         }
 
+        [Fact]
+        public async Task WritingText_When_ContentType_IsNull_SetPlainText()
+        {
+            HttpContext context = CreateRequest();
+            await context.Response.WriteAsync("Hello World");
+
+            Assert.Equal(ContentTypeConstants.PlainTextContentTypeWithCharset, context.Response.ContentType);
```

Comment:
I do like keeping this as a negative test case asserting that `context.Response.ContentType` is *not* set by `Response.WriteAsync(string)` once we move this logic out to RequestDelegateFactory.